apps/store/themes/store/partials/navigation.hbs

```diff
@@ -49,9 +49,9 @@
                             {{/if}}
                         {{/each}}
                     </div>
-                    <div id="tags-collapse" class="tags-collapse" align="center">
+                    <div id="tags-collapse" class="tags-collapse">
                         <p class="small">
-                            <span id="tags-collapse-text">Show all tags&nbsp;&nbsp;</span>
+                            <span id="tags-collapse-text" class="tags-collapse-span">Show all tags&nbsp;&nbsp;</span>
```

Comment:
AFAIK we cannot use CSS space property here. It defines the size of the space between two words. Here we need 2 spaces after a certain word. I referred to https://github.com/wso2/product-greg/blob/master/modules/es-extensions/store/extensions/app/greg_store/themes/store/partials/header.hbs#L43 and many other places we have followed the same pattern.